src/Hl7.Fhir.Specification/Specification/Snapshot/SnapshotGenerator.cs

```diff
@@ -890,6 +884,11 @@ private async T.Task mergeElement(ElementDefinitionNavigator snap, ElementDefini
                 // Now, recursively merge the children
                 await merge(snap, diff).ConfigureAwait(false);
 
+                // [MS 20220425] Make sure newly added contentReferences (from bases, types, and contentReferences) are absolute.
+                // Except when the it's a core profile (Derivation is not a Constraint)
+                if (snap.StructureDefinition.Derivation == StructureDefinition.TypeDerivationRule.Constraint)
+                    ensureAbsoluteContentReferences(snap, snap.StructureDefinition.BaseDefinition);
```

Comment:
Using `BaseDefinition` will I think not work when we have profiles on profiles (since BaseDefinition is then the Url of the parent profile, not the original core profile). Michel has some code in this file that actually finds the base url of the core type, not of the direct parent!